packages/client/src/generation/TSClient/Generatable.ts

```diff
@@ -1,18 +1,19 @@
+// TODO rename this to Generable (correct english term)
```

Comment:
https://en.wiktionary.org/wiki/generatable 😆

Reply:
Somehow it vexes me when I read it, I'm the able kind :laughing: 

https://en.wiktionary.org/wiki/generable

Reply:
But you have a point, this awful word is real

Reply:
"generable" is awful, it is missing the "te" part. "gener" is not even a word :p